docs/examples/onchain-nft.md

```diff
@@ -0,0 +1,361 @@
+---
+sidebar_label: OnChain NFT
+sidebar_position: 14
+---
+
+# Gear Non-Fungible Token
+
+### Introduction
```

Comment:
Pls keep only essential info about gNFT, for the rest - provide a link on existing page: https://wiki.gear-tech.io/examples/gnft-721

Reply:
Fixed.